VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitVideoFragment.java

```diff
@@ -216,6 +218,7 @@ public void setUserVisibleHint(boolean isVisibleToUser) {
         if ( ViewPagerDownloadManager.instance.inInitialPhase(unit) )
             return;
         if (isVisibleToUser) {
+            checkVideoStatus(unit);
```

Comment:
Hmm, we should probably separate out the error states from functionality in `checkVideoStatus()`, since we only want to re-show the messages from here.